pkg/cli/admin/release/extract.go

```diff
@@ -310,8 +334,32 @@ func (o *ExtractOptions) Run() error {
 	if o.ExtractManifests {
 		expectedProviderSpecKind := credRequestCloudProviderSpecKindMapping[o.Cloud]
 
+		include := func(m *manifest.Manifest) error { return nil } // default to including everything
```

Comment:
I am not sure how this will include everything. Or it is some code which will get changed in the final code?

Reply:
If later code calls this `include` function, it will always return a non-error `nil` for "include this manifest".  If the caller set `--included`, we replace the value of this `include` variable with the more restrictive implementation that errors for manifests that the CVO is not expected to include (e.g. "this manifest is tech-preview, but the current cluster is not, so I doubt it would be included").  I don't expect changes to that pattern before merging.  But I can add a more elaborate comment here or in the commit message if you think  that would help clarify for future developers?

Reply:
I like using an `error` because it's "I want to include this manifest.  Helper function, if you disagree tell me why."  I guess it doesn't allow the caller to conveniently fail-open ("Hmm, my helper function failed to form an opinion, so I guess I'll assume it had not concerns" is not possible), and instead "helper function failed to form an opinion" is just one of several possible reasons we might decide to exclude a manifest.  But that's somewhat academic, because there are no external calls in this "accept everything" lambda or in the `Manifest.Include` function.

Reply:
Yeah I see the usefulness, it's basically an annotated boolean. My concern is mostly that using `error` in this way is surprising for someone who's expecting idiomatic Go.

Just engaging in academic discussion here, I don't think you need to change the code :) It follows existing code which has a reasonable Godoc, so I'm good with it.